lib/from_openstudio/model.rb

```diff
@@ -129,6 +138,13 @@ def self.orphaned_shades_from_model(openstudio_model)
           site_transformation = shading_surface_group.siteTransformation
           shading_surface_group.shadingSurfaces.each do |shading_surface|
             result << Shade.from_shading_surface(shading_surface, site_transformation)
+            # Get shade construction if specified
+            shade_const_base = shading_surface.construction
+            unless shade_const_base.empty?
+              shade_const_obj = shade_const_base.get
+              shade_const = shade_const_obj.to_LayeredConstruction.get
+              $shade_construction << shade_const
```

Comment:
Again, I feel like this is a sign that something hasn't been coded in the right place when I see it copy/pasted 5 times.